plugins/BEdita/Core/src/Model/Action/GetObjectAction.php

```diff
@@ -54,6 +56,7 @@ public function execute(array $data = [])
         $conditions = [
             'deleted' => (int)!empty($data['deleted']),
         ];
+        $conditions += $this->statusCondition();
```

Comment:
Couldn't we implement and use a custom finder in `ObjectsTable` instead? Finders are inherited from child tables, so it should be alright to use regardless of the actual table being used.

Something like:
```php
public function findStatus(Query $query, array $options)
{
    $mode = reset($options);
    switch ($mode) {
        case 'on':
            return $query->where([$this->aliasField('status') => 'on']);

        case 'draft':
            return $query
                ->where(function (QueryExpression $exp) {
                    return $exp->in($this->aliasField('status'), ['on', 'draft']);
                });

        case 'off':
        case 'all':
            return $query;

        default:
            throw new BadFilterException('Invalid mode');
    }
}
```